utils/constants.ts

```diff
@@ -0,0 +1 @@
+export const DAILY_HADITH_KEY = 'dailyHadith'
```

Comment:
Why we need it

Reply:
Imagine, I removed it and used plain string.
1. How would you provide that you use the same key everywhere?
2. What would you do if the changed? Would you change the key manually everywhere?

Reply:
I just asked, relax